packages/zapp/console/src/components/WorkflowGraph/utils.ts

```diff
@@ -35,6 +35,11 @@ export function isExpanded(node: any) {
  * @returns     boolean
  */
 export const checkIfObjectsAreSame = (a, b) => {
+  // if one of the objects is null (undefined), objects can't be the same
+  if ((!a || !b) && a != b) {
```

Comment:
nit: if these are objects, then the equality check might not be enough.

How deep does the comparison need to be? would `lodash.isEqual(a, b) `be a better alternative?

Reply:
That's a good idea - we can completely replace this helper with `_.isEqual`, I was working on top of existing code and thought about using `_.has` which doesn't help with `null` values, but replacing the whole this with isEqual is even better!